de-de/git-de.html.markdown

```diff
@@ -180,6 +181,8 @@ Bringt alle Dateien im Arbeitsverzeichnis auf den Stand des Index oder des angeg
 ```bash
 # Ein Repo auschecken - wenn nicht anders angegeben ist das der master
 $ git checkout
+# Eine Datei auschecken - sie befindet sich dann auf dem aktuellen Stand im Repository
```

Comment:
I use this very often in my day-to-day work. When you edit I file and then recognize that this change was wrong or produces an error, I want to "revert" the file.